src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedSimpleProgramEntryPointSymbol.cs

```diff
@@ -240,5 +234,7 @@ internal override bool IsDefinedInSourceTree(SyntaxTree tree, TextSpan? definedW
 
             return false;
         }
+
+        public SyntaxNode ReturnTypeSyntax => CompilationUnit.Members.OfType<GlobalStatementSyntax>().First();
```

Comment:
>OfType<GlobalStatementSyntax>().First() [](start = 70, length = 39)

Consider using `.First(n => n.Kind == SyntaxKind.GlobalStatement)` to avoid allocations from `OfType<>()`.